### PR TITLE
Create a custom form questions reducer

### PR DESCRIPTION
**Pull request by ahmedhamodi**
Took inspiration from the custom form sections reducer that @xinhaoz made to make one for the questions. Let me know if anything needs to change!

### CODE REVIEW

File: src/Reducers/CustomFormQuestionsReducer.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+function customFormQuestionsReducer(
+  state: QuestionCard[],
+  action: EditQuestionsAction
+): QuestionCard[] {
+  return produce(state, (draftState: QuestionCard[]) => {
+    switch (action.type) {
+      case "LOAD":
+        return action.sections;
+      case "ADD_QUESTION":
+        draftState.splice(action.index + 1, 0, defaultNewQuestion);
```
Comment (GodGreg) [inline src/Reducers/CustomFormQuestionsReducer.ts:14]: I like

**Review by GodGreg — APPROVED** [review]
LGTM. Is move question and duplicate question handled separately?

**Comment by ahmedhamodi** [comment]
> LGTM. Is move question and duplicate question handled separately?

Yes. Whoever does move/duplicate will need to add an action for that in the reducer. I actually think I may be doing both of those... so I will do that!

@xinhaoz if you could review this, that would be great!

**Comment by GodGreg** [comment]
> > LGTM. Is move question and duplicate question handled separately?
> 
> Yes. Whoever does move/duplicate will need to add an action for that in the reducer. I actually think I may be doing both of those... so I will do that!
> 
> @xinhaoz if you could review this, that would be great!

I'd just ping her directly on slack :)

**Review by xinhaoz — APPROVED** [review]
LGTM!!!!

**Pull request merged**